<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjustesGrafico</class>
 <widget class="QDialog" name="AjustesGrafico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ajustes elementos del gráfico</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/diana.png</normaloff>:/images/diana.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:rgb(221, 221, 221)
}

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Ajustes de Radios</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxRInterior">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelRadioObjetivos">
          <property name="text">
           <string>Radio de objetivos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelRadioCircExterior">
          <property name="text">
           <string>Radio círculo exterior</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxRExterior">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelRadioCircInterior">
          <property name="text">
           <string>Radio círculo interior</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxRObjetivo">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Ajustes Colores objetivos Y FPS</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="labelColorObjSinMarcar">
          <property name="text">
           <string>Color sin marcar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxColorObjetivosSinMarcar"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelColorObjMarcados">
          <property name="text">
           <string>Color marcados</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxColorObjetivosMarcados"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelFPS">
          <property name="text">
           <string>FPS</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxFPS"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelUnidad">
       <property name="text">
        <string>Unidad para representación</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBoxUnidadGrafico"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCustomPlot" name="qCustomRepresentacionGrafico" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="qCustomPlotObjetivo" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: #5cb85c;
border: 2px solid #4cae4c;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: #47a447;
border: 2px solid #398439;
}

QPushButton:pressed {
border: 3px solid #398439;
}</string>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: rgb(255, 0, 0);
border: 2px solid;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: rgb(170, 0, 0);
border: 2px solid ;
}

QPushButton:pressed {
border: 3px solid rgb(102, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
